crates/subspace-gateway/src/commands/run/dsn.rs

```diff
@@ -8,18 +8,18 @@ use subspace_networking::{construct, Config, KademliaMode, Node, NodeRunner};
 /// Configuration for network stack
 #[derive(Debug, Parser)]
 pub(crate) struct NetworkArgs {
-    /// Multiaddrs of bootstrap nodes to connect to on startup, multiple are supported
+    /// Multiaddrs of DSN bootstrap nodes to connect to on startup, multiple are supported
     #[arg(long)]
-    pub(crate) bootstrap_nodes: Vec<Multiaddr>,
+    pub(crate) dsn_bootstrap_nodes: Vec<Multiaddr>,
```

Comment:
Let's not do this, the only reason we prefix it in `subspace-node` is because we have two networking stacks there (for now at least) and need to differentiate between them. In all other cases (farmer, bootstrap node, here) we don't need the prefix. This is part of why I was saying it should be more like farmer than like node.